client/src/components/App/Content/LandingPage/adminLandingPage/TimeRangeFilterCard/useTimeRangeFilterCard.ts

```diff
@@ -1,82 +1,88 @@
-import { useState } from 'react';
+import { useEffect, useState } from 'react';
 import { format } from 'date-fns';
+import { useHistory } from 'react-router-dom';
 
 import { TimeRange } from 'models/TimeRange';
 import timeRanges from 'models/enums/timeRanges';
 
-import AdminLandingPageFilters from '../AdminLandingPageFilters';
+import { HistoryState } from '../../InvestigationTable/InvestigationTableInterfaces';
 
 const dateMissingError = 'יש לבחור תאריך תחילה וסיום';
+const dateInvalid = 'תאריך לא תקין';
 const rangeError = 'טווח תאריכים צריך להיות תקין';
+export const defaultTimeRange = timeRanges[0];
+const customTimeRange = timeRanges[4];
 
-const useTimeRangeFilterCard = (props : Props) => {
+const useTimeRangeFilterCard = () => {
     
-    const {timeRangeFilter , setTimeRangeFilter, investigationInfoFilter, setInvestigationInfoFilter} = props;
-    const defaultTimeRange = timeRanges[0];
-    const customTimeRange = timeRanges[4];
-    const [isLoading, setIsLoading] = useState<boolean>(false);
     const [errorMes, setErrorMes] = useState<string>('');
+    const [timeRangeFilter, setTimeRangeFilter] = useState<TimeRange>(timeRanges[0]);
```

Comment:
use defaultTimeRange const
```suggestion
    const [timeRangeFilter, setTimeRangeFilter] = useState<TimeRange>(defaultTimeRange);
```